front/loadObjects.js

```diff
@@ -1,44 +1,118 @@
 'use strict';
 
-var Set = require('es6-set');
+var Map = require('es6-map');
 
 var rTree = require('./rTree.js');
-var buildingMap = require('./buildingMap.js');
+var buildingCache = require('./buildingCache.js');
+var unpackBuilding = require('./unpackBuilding.js');
+var infosFromMesh = require('./infosFromMesh.js');
+var meshFromId = require('./meshFromId.js');
+var LimitedEntryMap = require('./LimitedEntryMap.js');
+// var meshColor = require('../common/meshDefaultColor.js');
 
+function distance(a, b){
```

Comment:
Remove this function and use [Math.hypot](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot#Browser_compatibility) instead.